.goreleaser.yml

```diff
@@ -0,0 +1,17 @@
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+    goarch:
+      - amd64
+archives:
+  - format: binary
+    name_template: "{{ .Binary }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
```

Comment:
The goreleaser doesn't build images yet.
Have a look in https://github.com/vshn/k8up/blob/master/.goreleaser.yml for inspiration.